app/src/main/java/org/mozilla/reference/browser/NotificationManager.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
nit: There's a line break here in the license header that we do not have in other files (also `FirebasePush.kt`)